From cd8db76e18eea051d8a1cb0657efc47885226e1f Mon Sep 17 00:00:00 2001
From: Fei Yang <fei.yang@intel.com>
Date: Wed, 12 Sep 2012 10:40:51 -0700
Subject: [PATCH 28/78] HEADER: Include kconfig.h instead of autoconf.h

BZ: 58860

This is part of kernel 3.4 bring up work. There should be no impact on
building the current PSI mainline.
Linux 3.1.0 introduced kconfig.h which includes generated/autoconf.h,
drivers need to include kconfig.h instead.

Change-Id: If38a2876c8754ba35ba0ecdcd8c52845802a7708
Signed-off-by: Fei Yang <fei.yang@intel.com>
Reviewed-on: http://android.intel.com:8080/67591
Reviewed-by: Cohen, David A <david.a.cohen@intel.com>
Reviewed-by: De Cesco, Jonathan <jonathan.de.cesco@intel.com>
Tested-by: Ng, Cheon-woei <cheon-woei.ng@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77985
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/include/linux/compat-2.6.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/include/linux/compat-2.6.h b/wl12xx-compat/include/linux/compat-2.6.h
index c23e94a..3ccd051 100644
--- a/wl12xx-compat/include/linux/compat-2.6.h
+++ b/wl12xx-compat/include/linux/compat-2.6.h
@@ -2,7 +2,9 @@
 #define LINUX_26_COMPAT_H
 
 #include <linux/version.h>
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,33))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3,1,0))
+#include <linux/kconfig.h>
+#elif (LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,33))
 #include <generated/autoconf.h>
 #else
 #include <linux/autoconf.h>
-- 
1.7.9

